### ISSUE TITLE
Main article image has no alt text. Use aria-labelledby="*figcaption element ID*"

### ISSUE THREAD

**Issue by adamsilver** [Financial-Times/next-article#601]
Note:  Image resides in body - will likely need another transform etc. See article controller.

**Comment by andygnewman** [comment]
@adamsilver @i-like-robots 

Can do this within the transform/images, but only for images with an existing fig caption (not very widespread)

Can use the unique image url characters as the id for figcaption and insert an aria-labelledby attr on to the img tag with the same value, then remove the existing blank alt tag.

May breakdown if the same image is used multiple times in an article though - have seen this once, but fortunately only one image had a figcaption.

I've got a working codebase for this - if it sounds the right approach I can create a pr for it.
